Source/GUI/Dialog_TagEditor.cs

```diff
@@ -604,7 +604,7 @@ Rect RectForTag(int tIdx, bool full = false)
 
                 Widgets.DefIcon(descRect.LeftPart(.15f), def);
                 Widgets.Label(descRect.RightPart(.85f), def.LabelCap);
-                TooltipHandler.TipRegion(rowRect, def.DescriptionDetailed);
+                TooltipHandler.TipRegion(rowRect, () => def.DescriptionDetailed, def.index);
```

Comment:
DescriptionDetailed throws an exception if any stats references is missing, taking the entire window along with it.
Switching this to a function would mean it'd only throw an exception if you actually try to view the description of the specific entry.
Might be a better solution, this was just a "quick fix"